plugin/neotest.lua

```diff
@@ -0,0 +1,270 @@
+local cmd_completion_store = {
+  [""] = {
+    "attach",
+    "jump",
+    "output",
+    "output-panel",
+    "run",
+    "stop",
+    "summary",
+  },
+  jump = {
+    "next",
+    "prev",
+  },
+  ["output-panel"] = {
+    "open",
+    "close",
+    "toggle",
+  },
+  run = {
+    "file",
+    "last",
+  },
+  summary = {
+    "close",
+    "mark",
+    "open",
+    "toggle",
+  },
+  ["summary:mark"] = {
+    "clear",
+    "run",
+    "toggle",
+  },
+}
+
+local opt_completion_store = {
+  adapter = function(arg)
+    local part = arg:gsub("^adapter=", "")
+    return vim.tbl_filter(function(cmd)
+      return not not string.find(cmd, "^" .. part)
+    end, require("neotest").run.adapters())
```

Comment:
Use `require("neotest").state.adapter_ids()` instead, `neotest.run.adapters` is deprecated

Reply:
Updated